sdk/tests/js/webgl-test-harness.js

```diff
@@ -595,11 +616,15 @@ TestHarness.prototype.getTest = function(url) {
 TestHarness.prototype.reportResults = function(url, success, msg, skipped) {
   url = FilterURL(url);
   var test = this.getTest(url);
-  this.clearTimeout(test);
-  log((success ? "PASS" : "FAIL") + ": " + msg);
+  if (0) {
```

Comment:
Does removing this significantly reduce the ability to debug test failures from the logs?

Would adjusting the specific test `deqp/functional/gles3/vertexarrays/multiple_attributes.output.html` be warranted instead, if so?

Reply:
Once the harness is attached, the harness should choose if it wants to console.log or not. (and the main test runner UI should probably not spew to console.log)
Maybe the test should be adjusted, but that can be handled separately.

Reply:
OK.